builder/dockerfile/instructions/parse.go

```diff
@@ -250,7 +250,7 @@ func parseAdd(req parseRequest) (*AddCommand, error) {
 
 func parseCopy(req parseRequest) (*CopyCommand, error) {
 	if len(req.args) < 2 {
-		return nil, errAtLeastTwoArguments("COPY")
+		return nil, errNoDestinationArgument("COPY")
```

Comment:
I'm not sure if this change is correct here. I'm using Docker version 17.10.0-ce-rc1, build d866876.
```Dockerfile
FROM alpine
COPY
```
```
$ docker build .
Sending build context to Docker daemon  87.44MB
Error response from daemon: Dockerfile parse error line 2: COPY requires at least two arguments, but only one was provided. Destination could not be determined.
```